src/json_spirit/json_spirit_value.h

```diff
@@ -55,6 +55,14 @@ namespace json_spirit
         Value_impl( boost::uint64_t    value );
         Value_impl( double             value );
 
+        template < long=0 >
+          requires (!std::is_same_v< long, boost::int64_t >)
+        Value_impl( long               value );
+
+        template < long=0 >
+          requires (!std::is_same_v< unsigned long, boost::uint64_t >)
+        Value_impl( unsigned long      value );

```

Comment:
would you mind explaining the reason why we need this change in the commit message?

Reply:
Done!